setup.cfg

```diff
@@ -3,3 +3,12 @@ description-file = README.rst
 
 [bdist_wheel]
 universal=1
+
+[tool:pytest]
+norecursedirs= .tox
+
+[flake8]
+# E251 unexpected spaces around keyword / parameter equals
+# W391 blank line at end of file
+# E501 line too long (> 79 characters)
+ignore = E501,W391,E251
```

Comment:
This was default setup from cookiecutter project so I am using this. We can always expand it in future.

Ignore of the lines is simply because I did not want to deal with it. E251 is somewhere inside `setup.py` blank line at end of file, no idea why is that there and line too long - I dont know flake good enough to put a custom line in there. As you said we will swap to pylint in near future (but again need our plugin to parse for warnings and errors otherwise we will need to fix a lot there)